connman/doc/connman.conf.5.in

```diff
@@ -167,6 +167,13 @@ Enable regdomain to be changed along timezone changes. With this option set to
 true each time the timezone changes the first present ISO3166 country code is
 being read from /usr/share/zoneinfo/zone1970.tab and set as regdom value.
 Default value is false.
+.TP
+.BI OfonoConnectDelayTimeout= secs
```

Comment:
Are units of seconds granular enough here? I was wondering if msecs might be more appropriate. Internally you have accuracy of tenths of a second I guess.

If you think seconds are okay, then that's fine as far as I'm concerned too.

Reply:
I like the idea of your new approach. I thought your earlier implementation was fine too if I'm honest, but I can see the value in plugin-specific configuration values.